src/HW_models/NRF_DPPI.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef _NRF_HW_MODEL_DPPI_H
+#define _NRF_HW_MODEL_DPPI_H
+
+#include "nrfx.h"
+
+#ifdef __cplusplus
+extern "C"{
+#endif
+
+extern NRF_DPPIC_Type NRF_DPPI_regs;
+#define NUMBER_DPPI_CHANNELS 32
```

Comment:
You may use `DPPI_CH_NUM` instead. It is defined in nrf5340_network_peripherals.h